rest/rest-sqs/src/main/scala/org/elasticmq/rest/sqs/model/RedrivePolicyJson.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * All right reserved - Fronteer LTD
+ */
+package org.elasticmq.rest.sqs.model
+
+import spray.json.{DefaultJsonProtocol, JsNumber, JsObject, JsString, JsValue, JsonFormat, RootJsonFormat, deserializationError}
+
+
+object RedrivePolicyJson extends DefaultJsonProtocol {
+
+	implicit val format: JsonFormat[RedrivePolicy] = new RootJsonFormat[RedrivePolicy] {
+		def read(json: JsValue): RedrivePolicy = {
+			json.asJsObject.getFields("deadLetterTargetArn", "maxReceiveCount") match {
+				case Seq(JsString(deadLetterTargetArn), JsString(maxReceiveCount)) =>
+					RedrivePolicy(deadLetterTargetArn.split(":").last, maxReceiveCount.toInt)
+				case Seq(JsString(deadLetterTargetArn), JsNumber(maxReceiveCount)) =>
```

Comment:
I added this as the docs technically specify a number (but then oddly use a string in the examples).